### PR TITLE
Fix version inconsistencies in api/* (T through Z)

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Sub-PR of #3368 for the fixes of the files themselves.

### CODE REVIEW

**Comment by jpmedley** [comment]
@Elchi3 I want to make sure you don't merge this out from under me. Chrome 54 as the first version to have workers is wrong. I'm having trouble finding an exact version.

File: api/WindowOrWorkerGlobalScope.json [hunk 1]
```diff
@@ -1067,10 +1061,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/setInterval",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "54"
```
Comment (jpmedley) [inline api/WindowOrWorkerGlobalScope.json]: [This CL](https://chromium.googlesource.com/chromium/src/+/387fa9a4b0c7cbbcac73ab8bc201d5c36e366ab6) is as close as I can get to anything definitive. The time frame is Chrome 30 for desktop and Android. Webview should say  true.

File: api/WindowOrWorkerGlobalScope.json [hunk 2]
```diff
@@ -1175,10 +1169,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "54"
```
Comment (jpmedley) [inline api/WindowOrWorkerGlobalScope.json]: Also 30, 30, and true based on same CL.

File: api/WindowOrWorkerGlobalScope.json [hunk 3]
```diff
@@ -180,7 +180,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/caches",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "54"
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply api/WindowOrWorkerGlobalScope.json]: Chrome 4 it is!  I’ll revert the changes to this file and update the versions as you’ve stated.  :wink:

**Comment by vinyldarkscratch** [comment]
Review?  @Elchi3 @ddbeck

**Pull request merged**